platforms/hyperledger-indy/charts/indy-auth-job/templates/job.yaml

```diff
@@ -11,7 +11,9 @@ spec:
     spec:
       restartPolicy: OnFailure
       imagePullSecrets:
-        - name: "{{ $.Values.image.pullSecret }}"
+        {{ if and (ne $.Values.network.docker.password '') (ne $.Values.network.docker.username '') }}
```

Comment:
If you are thinking of passing the actual docker creds as values then its wrong. The value files should not contain any credentials. Rather check if image.pullSecret value is present or not in the helm chart and in the template files pass or not pass the pullSecret value based on your logic of docker creds are defined or not in the network.yaml